tools/bundle/Build.js

```diff
@@ -24,23 +24,28 @@ function writeTo(path, data, noJson) {
     });
 }
 
-function mergeEnvironments(configEnv, env) {
-    const conf = config.env[env];
+function mergeEnvironments(conf, env) {
+    if (!conf.env.hasOwnProperty(env)) return;
+    const envConf = conf.env[env];
 
     // Apply attributes from target `env` THAT HAVE NOT BEEN DEFINED IN CURRENT ENV into the target config
     // Priority is in index order i.e. if a config is defined at index 0 and 2, the value from index 0 will win
-    if (conf.inherits) {
-        inherit(conf.inherits, conf, configEnv);
-        delete conf.inherits;
+    if (envConf.inherits) {
```

Comment:
what do you think of updating the variables so that is more clear which `env` is the target environment?